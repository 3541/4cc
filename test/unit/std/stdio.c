#include <stdio.h>

int main(void) {
    size_t x = 23;
    int    y = -4;
    long   z = -19;

    printf("Greetings!\nSome numbers: %zu, %d, %ld.\n", x, y, z);
}
