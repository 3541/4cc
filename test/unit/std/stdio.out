Greetings!
Some numbers: 23, -4, -19.
